### PR TITLE
refactor: cleanup app.rs

### PR DESCRIPTION
**Pull request by nozwock**
## Refactor

- Remove explicitly stated data types where appropriate
- Replace complex code in favor of `unwrap_or_default()` for parsing of `String` to `u64`/`f64`
- Use mutable variable instead of its clone

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -253,20 +253,12 @@ fn send(event_type: &EventType) {
     }
 }
 
-fn parse_string_to_u64(string: String) -> u64 {
-    let mut unsigned_int: u64 = 0u64;
-    if !string.is_empty() {
-        unsigned_int = string.parse().unwrap();
-    }
-    unsigned_int
+fn parse_string_to_u64(string: &str) -> u64 {
```
[the remark this answers is not in the document]
Comment (MrTanoshii) [reply src/app.rs:256]: Sorry for the late review.
Yup, I 100% agree, the functions have lost their purpose.

Are you around?
Can you please remove the fns for `self.xxx_str.parse::<xxx64>().unwrap_or_default();`?

**Review by MrTanoshii — CHANGES_REQUESTED** [review]
The parsing functions can be removed in favor of directly using `parse().unwrap_or_default()`

There's `interval: u64` on L494 that can also be cleaned up of its data type

**Comment by MrTanoshii** [comment]
Contributes to #3

File: src/app.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (nozwock) [reply src/app.rs:256]: I'm around, was just thinking about what Cli project to work on...any idea I can think of is already done by someone in rust :crying_cat_face:

[the remark this answers is not in the document]
Comment (MrTanoshii) [reply src/app.rs:256]: Are you on Discord?
Same nickname?

[the remark this answers is not in the document]
Comment (nozwock) [reply src/app.rs]: Ya, nozwock#7587

**Comment by nozwock** [comment]
> There's `interval: u64` on L494 that can also be cleaned up of its data type

I was thinking of this rather
```rust
        let interval: u64 = (hr * 3600000) + (min * 60000) + (sec * 1000) + ms;
```
Let's leave that `u64` out just for being lil explicit?

**Comment by MrTanoshii** [comment]
> > There's `interval: u64` on L494 that can also be cleaned up of its data type
> 
> I was thinking of this rather
> 
> ```rust
>         let interval: u64 = (hr * 3600000) + (min * 60000) + (sec * 1000) + ms;
> ```
> 
> Let's leave that `u64` out just for being lil explicit?

yeah that works too

**Comment by nozwock** [comment]
> yeah that works too

Done

**Review by MrTanoshii — APPROVED** [review]

**Comment by MrTanoshii** [comment]
Thanks!

**Pull request merged**